nav2_bringup/bringup/params/nav2_params.yaml

```diff
@@ -270,8 +272,8 @@ planner_server:
           w_curve: 30.0 #3.0 # range 0 - 50
           w_dist: 0.0 #300 w/0 cost
           w_smooth: 15000.0 #15000
-          w_cost: 0.025
-          max_curve: 2.0         # translates to |delta angle| / |delta distance|, where distance is ~2x costmap resolution
+          w_cost: 2.5
```

Comment:
For right now, lets leave these as what they were before until we have hybrid sampling nodes available (this and the downsample_costmap to false)